### DISCUSSION TITLE
每次复习完之后都会在wiki生成log数据条目吗

### DISCUSSION THREAD

**Discussion by dongrentianyu** [oflg/Tidme#33] [General]
![image](https://github.com/user-attachments/assets/43ebd91b-cd43-414c-a310-e88bf4489a26)

有段时间没用了，重新用发现了会在wiki内生成log数据条目。如果每次复习都生成的话，那应该会有挺多数据条目吧。

**Comment by dongrentianyu** [comment]
![image](https://github.com/user-attachments/assets/d1bb78d5-0272-45ee-9133-d2b03ec98f53)

停止学习后这个条目就删除了，但log条目还在。

**Comment by oflg** [comment]
打算log用来做日程热力图。

每天的log最多一条（不复习就没有）

**Reply by dongrentianyu** [reply]
那好吧。我还是把卡片复习单独列出来。不然每天复习的数据感觉会很影响存储。

**Comment by linonetwo** [comment]
目前的效果是，复习时不会更新时间戳，仅编辑卡片内容才更新，这样不会在「最近」里刷屏，可以保留真正的「编辑」的时间。

但是会更新最后复习时间，用于作图。

而如果修改了卡片，则就应该更新「编辑」的时间。
